phpstan.neon

```diff
@@ -692,6 +692,7 @@ parameters:
         - '#Class with base "Application" name is already used in "(.*?)\\Symfony\\Component\\Console\\Application", "Symfony\\Component\\Console\\Application", "Rector\\Core\\Console\\Application"\. Use unique name to make classes easy to recognize#'
         - '#Class with base "FileNode" name is already used in "PHPStan\\Node\\FileNode", "Rector\\Core\\PhpParser\\Node\\CustomNode\\FileNode"\. Use unique name to make classes easy to recognize#'
         - '#Class cognitive complexity for "BetterStandardPrinter" is 52, keep it under 50#'
+        - '#Class cognitive complexity for "NullsafeOperatorRector" is 53, keep it under 50#'
```

Comment:
These should not be ignored, but rather extract a new service with some common logic

Reply:
I will try

Reply:
implemented, moved check if cond using assign identical/not identical to IfManipulator service

Reply:
I moved NullSafe create process to new service named `NullsafeManipulator`